desktop-fonts/maple-font/autobuild/build

```diff
@@ -0,0 +1,10 @@
+abinfo "Installing font file..."
+
+install -Dm644 "$SRCDIR"/*.ttf \
```

Comment:
Verbosity here, please see the [Styling Manual](https://wiki.aosc.io/developer/packaging/package-styling-manual/).